app/classifier/tasks/survey/choice.cjsx

```diff
@@ -97,7 +97,7 @@ module.exports = React.createClass
                   <ImageFlipper images={@props.task.images[filename] for filename in otherChoice.images} />
                   <Markdown content={choice.confusions[otherChoiceID]} />
                   <div className="survey-task-choice-confusion-buttons" style={textAlign: 'center'}>
-                    <button type="submit" className="major-button identfiy">Dismiss</button>
+                    <button type="submit" autoFocus={true and otherChoice.images.length < 2} className="major-button identfiy">Dismiss</button>
```

Comment:
I'm going to have to revisit the logic for how autofocus is assigned when the image slideshow isn't present, so I'll leave changing this to a future PR.